template.yaml

```diff
@@ -274,3 +282,21 @@ Resources:
       Name: /NVA/Athena/QueriesBucketName
       Type: String
       Value: !Ref NvaAthenaQueriesBucket
+
+  SearchInfrastructureApiUriParameter:
+    Type: AWS::SSM::Parameter
+    Properties:
+      DataType: text
+      Description: Parameter containing the URI to external search infrastructure API (SWS).
+      Name: /NVA/Search/InfrastructureApiUri
+      Type: String
+      Value: !Ref SearchInfrastructureApiUri
+
+  SearchInfrastructureAuthUriParameter:
+    Type: AWS::SSM::Parameter
+    Properties:
+      DataType: text
+      Description: Parameter containing the URI to auth for external search infrastructure API (SWS).
+      Name: /NVA/Search/InfrastructureAuthUri
+      Type: String
```

Comment:
I am not sure that these parameters are necessary.  Are they going to be used by any service? If yes maybe we should add a comment